sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/UpdateCenterDeserializer.java

```diff
@@ -235,7 +235,13 @@ private void parsePluginReleases(Properties p, Sonar sonar, String pluginKey, Pl
     }
   }
 
-  private Release parsePluginRelease(Properties p, Sonar sonar, String pluginKey, Plugin plugin, boolean isPublicRelease, boolean isArchivedRelease, String pluginVersion) throws IllegalArgumentException {
+  private Release parsePluginRelease(Properties p
+                                    , Sonar sonar
```

Comment:
Seems strange to me to have comma upfront. Not sure our coding conventions are precise enough on this topic, but that's the first time I see that in our codebase.